frontend/src/components/Playback/MiniVideoPlayer.vue

```diff
@@ -7,13 +7,15 @@
       :scrim="false"
       scroll-strategy="none"
       content-class="minimized-overlay"
+      :close-on-back="false"
```

Comment:
```suggestion
      close-on-back="false"
```

The same thing applies to all the other occurrences. Doesn't it work this way for you?

Reply:
As for creating JOverlay I am interested but I have time commitment issues 😓  I imagine that our need for functionality from it will be limited and hopefully not a very big deal. I'm just going to prioritize that mini->fullscreen pausing bug for now and then I can circle around.
But I hope this PR can still stand on its own. It fixes a really annoying issue with some tiny code changes :)